bigquery_etl/cli/query.py

```diff
@@ -587,7 +587,7 @@ def backfill(
         ctx.invoke(
             generate_all,
             output_dir=ctx.obj["TMP_DIR"],
-            ignore=["derived_view_schemas", "stable_views"],
```

Comment:
Solved, thank you

<img width="1210" alt="Screen Shot 2022-05-09 at 13 05 55" src="https://user-images.githubusercontent.com/30448600/167397919-7a340dca-0888-4a0b-9b3d-f0107dc8575a.png">